File: bench/c/fib.c
#include <stdio.h>

static int fib(int n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

int main(void) {
    int r = fib(40);
    printf("fib(40) = %d\n", r);
    return 0;
}

File: bench/go/fib.go
package main

import "fmt"

func fib(n int32) int32 {
	if n < 2 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	r := fib(40)
	fmt.Printf("fib(40) = %d\n", r)
}
